src/standalone/Vegetation/Canopy.jl

```diff
@@ -506,7 +507,7 @@ function ClimaLSM.make_update_aux(
             canopy_surface_fluxes(canopy.atmos, canopy, Y, p, t)
         transpiration .= canopy_transpiration
         # Transpiration is per unit ground area, not leaf area (mult by LAI)
-        fa[n_stem + n_leaf] .= PlantHydraulics.transpiration_per_ground_area(
+        fa.:($i_end) .= PlantHydraulics.transpiration_per_ground_area(
```

Comment:
not totally sure why we needed the $ here, either, since this is not in a loop.

Reply:
We can/should probably document how this is working, but it looks fine to me

Reply:
I agree that we should document how the indexing works now, Ill add some comments!

Reply:
I'm also confused about why we use a `$` here - shouldn't it just be `fa.:i_end`?

Reply:
Im not sure. the `:i_end` notation says to look for the field associated with the symbol `i_end` if I understand correctly, and so it doesnt work here.
```
julia> i_end = n_stem+n_leaf
11
julia> fa.:i_end
ERROR: Invalid field name i_end for type NTuple{11, Float64}
```

This doesnt work either:
```
julia> fa.:($(n_stem+n_leaf))
ERROR: syntax: invalid syntax "fa.((n_stem + n_leaf))" around REPL[104]:1
Stacktrace:
 [1] top-level scope
   @ REPL[104]:1
```

Reply:
The integer needs `$` to be interpolated otherwise the symbol (used to call `getproperty(fa, ::Symbol)`) is `:i_end`